entries/entry02.md

```diff
@@ -1,7 +1,32 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 0

Use markdown! (especially for links). Any lists? Code?